Source/Scene/Expression.js

```diff
@@ -212,7 +202,7 @@ define([
     Expression.prototype.evaluateColor = function(frameState, feature, result) {
         ScratchStorage.reset();
         var color = this._runtimeAst.evaluate(frameState, feature);
-        return Color.clone(color, result);
+        return Color.fromCartesian4(color, result);
```

Comment:
Yes, agreed.  It is fine for the Cesium implementation to map `vec4` to Cesium's `Color` when we know it is the `color` property.  This is an implementation detail and does not affect the spec.